src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -33,6 +33,7 @@ public class DragAndDrop : BaseHandle
         public delegate void SourceEventHandler(DragSourceEventType sourceEventType);
```

Comment:
Could you add `[EditorBrowsable(EditorBrowsableState.Never)]` tag here and 35 line, too?

Reply:
Please ignore my comment :)